### ISSUE TITLE
Choosing NONE for cellular modem results in 404 during lookup for modem_on.py

### ISSUE THREAD

**Issue by shollifield** [farmjenny/Farm_Jenny_Installer#1]
```
--2022-12-02 04:24:14--  https://raw.githubusercontent.com/farmjenny/Farm_Jenny_Installer/master/installer/util/NONE/modem_on.py
Resolving raw.githubusercontent.com (raw.githubusercontent.com)... 185.199.110.133, 185.199.111.133, 185.199.108.133, ...
Connecting to raw.githubusercontent.com (raw.githubusercontent.com)|185.199.110.133|:443... connected.
HTTP request sent, awaiting response... 404 Not Found
2022-12-02 04:24:15 ERROR 404: Not Found.
```

Appears as though there is no 'NONE' folder.

**Comment by rcrouthamel** [comment]
Hi Sam, sorry for the inconvenience.  I'll fix the installer later today/tonight.  In the meantime, there shouldn't be any harm in selecting one of the modems instead.  I'm assuming you're using the HAT without a modem.

**Comment by rcrouthamel** [comment]
Resolved in 90b0e030c045bf0f95431d5f37311d222f487a43
